### ISSUE TITLE
Weird types when batching `FindMany`

### ISSUE THREAD

**Issue by codemaster138** [Brendonovich/prisma-client-rust#208]
I'm working on a server that uses Prisma Client Rust to interact with its database. One of its endpoints needs to run multiple `find_many` queries, so I'm trying to batch them into a single request. My code looks something like this:

```rust
use crate::prisma::my_object::{Data, FindMany};

let db: &PrismaClient = todo!();

let queries: (FindMany, FindMany) = todo!();

let results = db
	._batch(queries)
	.await
	.unwrap(); 
```

Now, I would expect `results` to be a tuple with one item corresponding to each item in `queries` and containing a `Vec<Data>` since each `FindMany` query should return a `Vec<Data>`.

However, the actual type of `results` is `(Data, Data)`.

Is my code wrong, or is this another code generation bug?

**Comment by Brendonovich** [comment]
Not a codegen bug but a library bug, is an easy fix

**Comment by Brendonovich** [comment]
@codemaster138 should be fixed in `rev = "999f772cfd4d5185b451ec8d5a018b2f5f7b17d7"`

**Comment by codemaster138** [comment]
Yep, seems to be working
